src/Sulu/Bundle/HttpCacheBundle/Tests/Unit/EventSubscriber/InvalidationSubscriberTest.php

```diff
@@ -28,29 +30,22 @@
 use Sulu\Component\DocumentManager\Event\RemoveEvent;
 use Sulu\Component\DocumentManager\Event\UnpublishEvent;
 use Sulu\Component\DocumentManager\Metadata;
-use Sulu\Component\HttpCache\EventSubscriber\InvalidationSubscriber;
-use Sulu\Component\HttpCache\HandlerInvalidatePathInterface;
-use Sulu\Component\HttpCache\HandlerInvalidateStructureInterface;
 use Sulu\Component\Webspace\Manager\WebspaceManagerInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestStack;
 
 class InvalidationSubscriberTest extends \PHPUnit_Framework_TestCase
 {
+    const DOCUMENT_UUID = '743389e6-2ac5-4673-9835-3e709a27a03d';
```

Comment:
I think it is not a good idea to introduce constants like this... This test wouldn't fail, if we hard code this uuid in the code. I would use different uuids in every test case.